examples/hub_demo/hub_demo.ino

```diff
@@ -238,6 +239,83 @@ byte getconfdescr( byte addr, byte conf )
   }//while( buf_ptr <=...
   return ( rcode );
 }
+
+// function to get all string descriptors
+byte getallstrdescr(uint8_t addr)
+{ 
+  byte rcode;
+  Usb.Task();
+  if( Usb.getUsbTaskState() >= 0x80 ) {  // state configuring or higher
+    USB_DEVICE_DESCRIPTOR buf; 
+    rcode = Usb.getDevDescr( addr, 0, 0x12, ( uint8_t *)&buf );
```

Comment:
Could you replace 0x12 with DEV_DESCR_LEN, so it will become clearer. There is one more here as well: https://github.com/stumpja/USB_Host_Shield_2.0/blob/master/examples/hub_demo/hub_demo.ino#L119. It would be nice if you could fix that as well while you at it.